compose-builder/get-api-gateway-token.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+mkdir keys
+openssl genpkey -algorithm RSA -out keys/rsa4096.key -pkeyopt rsa_keygen_bits:4096
+openssl rsa -in keys/rsa4096.key -pubout -out keys/rsa4096.pub
+JWT=`docker run --rm -it -e KONGURL_SERVER=kong --network edgex_edgex-network --entrypoint "" \
```

Comment:
During testing, docker is outputting a \r that is being captured in the JWT variable incorrectly.

Instead use
```
ID=`uuidgen`
```

and pass that in to the proxy jwt command below, instead of capturing the output of proxy adduser.

Reply:
Done